web/src/utils.js

```diff
@@ -49,11 +49,23 @@ function copyToClipboard(text) {
   return false;
 }
 
+function getDandisetContact(dandiset) {
```

Comment:
Do you anticipate this being reused in more places than just the Dandiset landing page?

Reply:
That was my reasoning in having it in `utils.js`. For example, in the redesign of the [DLP](https://www.figma.com/proto/kEHOgqU0rcOIyCYIDlpSMV/DANDI-UI?scaling=min-zoom&node-id=52%3A656) the contact is shown.